projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -416,8 +419,9 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
       this.localTableConfig.searchConfig.searchTerm = searchTerm;
       if (!this.isServerMode()) {
         this.performSearch(searchTerm.toLowerCase());
+      } else {
```

Comment:
Should we emit this regardless of whether it's serverMode or not. Everywhere else regardless of whether it's the server mode or not I believe we're still emitting that something changed.

Reply:
We actually should not. It will duplicate the event because we call `tableChangeOutcome` within `performSearch`.